test/binomial.jl

```diff
@@ -13,6 +13,9 @@ using HypothesisTests: default_tail
     @test_ci_approx confint(t, method=:wald) (0.22871819521037956, 0.43794847145628707)
     @test_ci_approx confint(t, tail=:left, method=:wald) (0.0, 0.42112912485444692)
     @test_ci_approx confint(t, tail=:right, method=:wald) (0.24553754181221971, 1.0)
+    @test_ci_approx confint(t, method=:waldcc) (0.22230793880012298, 0.4443587278665436)
```

Comment:
Out of curiosity, how did you generate the comparison values?

Reply:
This is original output of function. 
I checked this with R `DescTools` package.
```
library(DescTools)
ci <- BinomCI(26,78, method = "waldcc")
sprintf("%.14f",ci[2])
sprintf("%.14f",ci[3])
```

Result:

```
[1] "0.22230793880012"
[1] "0.44435872786654"
```

15-17 digits not validated :(